dotnet/src/SemanticKernel.Abstractions/KernelExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Orchestration;
+
+namespace Microsoft.SemanticKernel;
+
+public static class KernelExtensions
```

Comment:
Classic example of "just one more thing"... expanding the scope of a PR beyond its original intent. I've deleted this class to avoid introducing yet more public methods until we can finalize.

Reply:
Deleted this file.